v0/destinations/active_campaign/transform.js

```diff
@@ -131,6 +131,48 @@ const customTagProcessor = async (message, category, destination) => {
     res.data.tags.map(t => {
       storedTags[t.tag] = t.id;
     });
+    const promises = [];
+    if (
+      res.data.meta &&
+      res.data.meta.total &&
+      parseInt(res.data.meta.total, 10) > 100
+    ) {
+      for (
+        let i = 0;
+        i < Math.floor(parseInt(res.data.meta.total, 10) / 100);
+        i++
+      ) {
+        try {
+          const resp = axios.get(
+            `${destination.Config.apiUrl}${
+              category.tagEndPoint
+                ? `${category.tagEndPoint}?limit=100&offset=${100 * (i + 1)}`
+                : ""
```

Comment:
if without `category.tagEndPoint` it is not possible to use `limit` or `offset` in query params, can we check it's availability previously? I think then the line of code will be a bit easy to read.

Reply:
We can actually remove the ternary condition as `tagEndPoint` will always be present in category

Reply:
@yashasvibajpai Check once